tests/cli/demultiplex/test_create_sample_sheet.py

```diff
@@ -37,10 +40,10 @@ def test_create_sample_sheet_no_run_parameters(
         return_value=lims_novaseq_bcl2fastq_samples,
     )
 
-    # GIVEN a demux API context
-    demux_api: DemultiplexingAPI = sample_sheet_context.demultiplex_api
-    demux_api.flow_cells_dir: Path = tmp_flow_cells_directory_no_run_parameters.parent
-    sample_sheet_context.demultiplex_api_: DemultiplexingAPI = demux_api
+    # GIVEN that the flow cell directory is in the context
```

Comment:
What does context mean? Maybe change to something more descriptive

Reply:
fixed